modules/telco-troubleshooting-general-query-cluster.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/troubleshooting/telco-troubleshooting-general-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="telco-troubleshooting-general-query-cluster_{context}"]
+= Querying your cluster
+
+Get information about your cluster.
+
+.Procedure
+
+. Switch into a project by running the following command:
++
+[source,terminal]
+----
+oc project <project_name>
+----
+
+. Query your cluster version, cluster Operator, and node within that namespace by running the following command:
++
+--
+[source,terminal]
+----
+oc get clusterversion,clusteroperator,node
+----
+
```

Comment:
I'm a little confused here about how you run one command and get two separate outputs 🤔

Reply:
I made an editing mistake, it should be one output. Corrected now :)